model/system/metrics.yaml

```diff
@@ -102,7 +102,7 @@ groups:
     unit: "By"
     attributes:
       - ref: system.paging.state
-      - ref: system.device
+      - ref: linux.mdraid.device.name
```

Comment:
This seems unintentional; `system.device` is probably still the correct attribute for these.